annotations/src/main/java/mindustry/annotations/misc/LoadRegionProcessor.java

```diff
@@ -81,6 +81,7 @@ public void process(RoundEnvironment env) throws Exception{
             method.endControlFlow();
         }
 
+        method.addCode("mindustry.graphics.Drawf.textureResize.forEach(e -> { $T.atlas.find(e.key).scale = e.value; });", Core.class);
```

Comment:
You cannot use `forEach` in the Mindustry codebase as it is not supported on Android/iOS, use `each` instead.

This whole textureResize map thing is pretty weird, I don't have any better ideas yet but I would like to try some other approaches when this is merge-able.

Reply:
For overrides wouldn't it also make sense to determine the scale factor based on how much larger than the original it is?

Reply:
> For overrides wouldn't it also make sense to determine the scale factor based on how much larger than the original it is?

I think its a bad idea since it allows for multiple resolutions within a single texture pack way too easily and people will just not care anymore about the size of their sprites

Reply:
> I think its a bad idea since it allows for multiple resolutions within a single texture pack way too easily and people will just not care anymore about the size of their sprites

I guess thats fair..